src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -95,6 +108,14 @@ const NodeDetails: React.FC<INodeProps> = ({ expandDrawer }) => {
     pluginParameters,
   ]);
 
+  const path = `${baseUrl}plugins/instances/${id}/files/`
```

Comment:
String concatenation does work, but it's better to rely on the API to tell us what the correct link is instead of hard-coding it.

I'm more familiar with the backend Collection+JSON API than I am with the front-end code. Knowing that the response on `/api/v1/plugins/instances/N/` is going to return a link to `/api/v1/plugins/instances/N/files/`, the value for `path` that you want already exists as a variable in some object somewhere, and you just need to find it.

My first attempt was to just dig around and here's what I got:

```javascript
Object.fromEntries(selectedPlugin.collection.items[0].links.map((o: any) => [o.rel, o.href])).files
```

Next, I consulted the source code of [@fnndsc/chrisapi](https://www.npmjs.com/package/@fnndsc/chrisapi) and tried using its helper methods:

```javascript
import { Collection } from "@fnndsc/chrisapi";
// -- snip --
Collection.getLinkRelationUrls(selectedPlugin.collection.items[0], 'files')[0]
```

The code is pretty complicated but in my opinion, it's preferable to hard-coding. Rather, the fault lies in the implementation of \@fnndsc/chrisapi for not having a getter helper function for this piece of data.

Reply:
Good insight @jennydaman 
We have made an update, please look at it. Thank you.